pkiCert: CA chain field missing

https://github.com/hashicorp/consul-template/pull/1591 allows us to get `Cert`, `Key` and `CA` but there's no `ca_chain` key in the `.Data` object.
Could you please add it?

The template that I'm using:
```
{{- with pkiCert "pki/issue/foo" "common_name=bar.tld" "alt_names=localhost" "ip_sans=127.0.0.1" "ttl=720h"}}
{{.Data.Cert }}
{{.Data.ca_chain }}
{{.Data.Key }}
{{ end }}
```
The error that I'm getting:
```
[ERROR] template.server: template server error: error="/root/test.pem.tpl: execute: template: :16:8: executing \"\" at <.Data.ca_chain>: can't evaluate field ca_chain in type dependency.PemEncoded"

```

I'm using the latest version of Consul Template / Vault Agent.

--- a/docs/templating-language.md
+++ b/docs/templating-language.md
@@ -624,6 +624,18 @@ Cert Authority: {{ .CA }}
 {{ end }}
 ```
 
+If you need the complete CA+Chain in the same file as your Cert and Key use the standard `secret` to include it, but note the CA in the pkiCert return should be ignored as it would be redundant and upon reload might grab one of the chain certs. Something like this.
+
+```golang
+{{ with pkiCert "pki/issue/my-domain-dot-com" "common_name=foo.example.com" }}
+Private Key: {{ .Key }}
+Cert Authority: {{ .CA }}
+{{ end }}
+{{ with secret "pki/cert/ca_chain" }}
+CA+Chain: {{ .Data.ca_chain }}
+{{ end }}
+`
+
 ### `service`
 
 Query [Consul][consul] for services based on their health.

